### PR TITLE
table/table: set default value when write only column is null

### PR DESCRIPTION
**Pull request by coocood**
The test cover this issue need some work, will be added in next PR.

### CODE REVIEW

**Comment by coocood** [comment]
@shenli @zimulala PTAL

File: table/tables/tables.go [hunk 1]
```diff
@@ -201,7 +201,14 @@ func (t *Table) UpdateRecord(ctx context.Context, h int64, oldData []types.Datum
 	// Compose new row
 	t.composeNewData(touched, currentData, oldData)
 	colIDs := make([]int64, 0, len(t.writableCols()))
-	for _, col := range t.writableCols() {
+	for i, col := range t.writableCols() {
+		if col.State != model.StatePublic && currentData[i].IsNull() {
```
Comment (shenli) [inline table/tables/tables.go]: If col.State is writeOnly, should we add default value?

**Comment by shenli** [comment]
LGTM

**Comment by zimulala** [comment]
LGTM

**Pull request merged**